be/src/storage/lake/delta_writer.cpp

```diff
@@ -118,6 +118,8 @@ class DeltaWriterImpl {
 
     std::vector<std::string> files() const;
 
+    std::unordered_map<std::string, size_t> files_to_size() const;
```

Comment:
Change `std::vector<std::string> files()` to `std::vector<DirEntry> files()` and remove this method

Reply:
`FileInfo` can be defined in fs.h as follow:
```
struct FileInfo {
    std::string path;
    std::optional<int64_t> size;
};
```

Reply:
I've submit a PR that contains the `FileInfo`: https://github.com/StarRocks/starrocks/pull/37358